### PR TITLE
ember-cli-update to 2.16

### PR DESCRIPTION
**Pull request by Dhaulagiri**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,22 +4,28 @@ node_js:
...
-  - EMBER_TRY_SCENARIO=ember-beta
-  - EMBER_TRY_SCENARIO=ember-canary
-  - EMBER_TRY_SCENARIO=ember-default
+  global:
+    # See https://git.io/vdao3 for details.
+    - JOBS=1
+  matrix:
+    # we recommend new addons test the current and previous LTS
+    # as well as latest stable release (bonus points to beta/canary)
+    - EMBER_TRY_SCENARIO=ember-lts-2.8
```
Comment (raycohen) [inline .travis.yml]: we have been thinking of dropping support for older ember versions, but that would ideally be separate from this PR. If we retain all the scenarios we were previously, do the tests still pass with the rest of these updates?

File: package.json [hunk 2]
```diff
@@ -20,32 +20,31 @@
...
     "ember-cli-test-loader": "^1.1.0",
-    "ember-cli-uglify": "^1.2.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-cli-uglify": "^2.0.0",
+    "ember-disable-prototype-extensions": "^1.1.2",
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
-    "ember-source": "~2.13.0",
-    "ember-suave": "4.0.1",
```
Comment (raycohen) [inline package.json]: was removing ember-suave intentional? can we keep it?

File: .travis.yml (hunk 1 above)
Comment (Dhaulagiri) [reply .travis.yml]: I suspect they will work.  I'll put them back and see how things go.

File: package.json (hunk 2 above)
Comment (Dhaulagiri) [reply package.json]: ember-suave is unmaintained and  based on jscs whereas ember-cli has moved on to eslint for linting.  I can try adding https://github.com/DockYard/eslint-plugin-ember-suave instead and see if that has a similar effect.

**Comment by Dhaulagiri** [comment]
@raycohen I made those updates and the suite is still green.  Let me know if you want me to take a look at anything else.

**Comment by raycohen** [comment]
@Dhaulagiri I hadn't realized that ember-suave is unmaintained. Thanks for adding the plugin. This is great!

**Pull request merged**